pkg/agent/utils.go

```diff
@@ -194,6 +195,33 @@ func getBase64EncodedGzippedCustomScript(csFilename string, config *datamodel.No
 	return getBase64EncodedGzippedCustomScriptFromStr(csStr)
 }
 
+func removeComments(b []byte) []byte {
+	var contentToKeep []string
+	lines := strings.Split(string(b), "\n")
```

Comment:
should we be using bufio ScanLines or similar?

Reply:
this will add _some_ latency to the call from RP to get node bootstrapping right?

I'd imagine the difference in terms of latency between the 2 methods is most likely negligible however

Reply:
We could only be sure once we run a benchmark, as my friend Bill K says